polling/coincap.go

```diff
@@ -55,52 +58,45 @@ func CallCoinCap(config *config.Config) (CoinCapResponse, error) {
 	return CoinCapResponse, err
 }
 
-func HandleCoinCap(response CoinCapResponse, peg *PegAssets) {
+func HandleCoinCap(response CoinCapResponse, peg PegAssets) {
 
 	var timestamp = response.Timestamp
 
 	for _, currency := range response.Data {
-		if currency.Symbol == "XBT" || currency.Symbol == "BTC" {
+		switch currency.Symbol {
+		case "BTC", "XBT":
 			value, err := strconv.ParseFloat(currency.PriceUSD, 64)
-			peg.XBT.Value = Round(value)
-			peg.XBT.When = timestamp
+			peg["XBT"] = PegItem{Value: Round(value), When: timestamp}
 			if err != nil {
 				continue
 			}
-		} else if currency.Symbol == "ETH" {
+		case "BCH", "XBC":
```

Comment:
Coincap api can return different tickers?

Reply:
I believe there is a ticker difference from coincap to what we want. But that logic was there before I touched it, I didn't change much actual logic, just making the swap.